NEWS.md

```diff
@@ -1,5 +1,8 @@
 # teal.widgets 0.1.1.9013
 
+### Breaking changes
+* `panel_group` and `panel_item` are no longer offer to be optionally a shiny input.
```

Comment:
I think this needs to be a little clearer - it's the css class name rather than the panel group itself?